### DISCUSSION TITLE
Misleading macOS code sign doc

### DISCUSSION THREAD

**Discussion by setoelkahfi** [tauri-apps/tauri#11977] [Q&A]
The macOS codesign doc says:

`Code signing on macOS requires an [Apple Developer](https://developer.apple.com/) account which is either paid (99$ per year) or on the free plan`

Isn't it misleading? I need a paid account to create a signing certificate.

**Comment by FabianLars** [comment]
https://github.com/tauri-apps/tauri-docs/issues/2972
